### ISSUE TITLE
Zend\Mail: email parsing doesn't support base64 encoded comma inside name

### ISSUE THREAD

**Issue by echampet** [zendframework/zendframework#7363]
Hi

#6267 isn't enough. 
in https://github.com/zendframework/zf2/blob/master/library/Zend/Mail/Header/AbstractAddressList.php#L44
str_getcsv should happen before iconv_mime_decode

An exemple of "broken" header 
```
From: =?utf-8?B?QUFBLCB0ZXN0IHdpdGggw6nDoCTDuQ==?= <aaaa@bbbb.com>
```

**Comment by samsonasik** [comment]
PR ?

**Comment by echampet** [comment]
@samsonasik, I'm working on many patches right now

**Comment by GeeH** [comment]
This issue has been moved from the `zendframework` repository as part of the bug migration program as outlined here - http://framework.zend.com/blog/2016-04-11-issue-closures.html
New issue can be found at: https://github.com/zendframework/zend-mail/issues/101
